controllers/nodehealthcheck_controller_test.go

```diff
@@ -813,6 +814,38 @@ var _ = Describe("Node Health Check CR", func() {
 
 			})
 
+			When("a Succeded remediation times out", func() {
```

Comment:
IIUC the purpose of the timeout annotation is to single the remediator to stop working in order to prevent a conflict with the next remediator.
Because of that I think the test should be int the context of   "with multiple escalating remediations"

Reply:
If the `Succeeded` condition is True, shoudln't have the operator already stopped working?

Reply:
Indeed it has, but setting the annotation modify's the remediation operator's CR and triggers another Reconcile.

I guess what I mean is that in case there is only one remediation (as in this case) placing timeout annotation is less significant (so I prefer to test the "more" significant scenario).
- either the remediator is already done (similar to what you've suggested with multi remediators)
- or it is still working, but in any case it will not prevent a conflict with another remediator

But definitely a Nit.

Reply:
I see, I'll try to add the test to the other context too